.cirrus.yml

```diff
@@ -1,3 +1,29 @@
+task:
+  auto_cancellation: true
+
+  env:
+      PY_VER: "3.6"
+
+  name: Black Formatting
+  container:
+      image: python:latest
+  env:
+      PATH: $HOME/conda/bin:$PATH
+  conda_script:
```

Comment:
this was only here to install python but we're just using a container version so it's no longer necessary and should be removed